client/src/components/MovieListItem/MovieListItem.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import './MovieListItem.css';
+import { Link } from 'react-router-dom';
+
+const MovieListItem = ({movie}) => {
+
+  const img_path = 'https://image.tmdb.org/t/p/w300';
+  const [prob , setProb] = useState(false)
```

Comment:
Is there a better name for 'prob'? I'm not sure exactly what this 'prob' means here.